src/raw/requests.rs

```diff
@@ -14,27 +13,17 @@ use kvproto::tikvpb::TikvClient;
 use std::mem;
 use std::sync::Arc;
 
-pub(super) struct RawGet {
-    pub key: Key,
-    pub cf: Option<ColumnFamily>,
-}
-
-impl KvRequest for RawGet {
+impl KvRequest for kvrpcpb::RawGetRequest {
     type Result = Option<Value>;
-    type RpcRequest = kvrpcpb::RawGetRequest;
     type RpcResponse = kvrpcpb::RawGetResponse;
     type KeyData = Key;
     const REQUEST_NAME: &'static str = "raw_get";
-    const RPC_FN: RpcFnType<Self::RpcRequest, Self::RpcResponse> = TikvClient::raw_get_async_opt;
+    const RPC_FN: RpcFnType<Self, Self::RpcResponse> = TikvClient::raw_get_async_opt;
 
```

Comment:
I think it would be good to have ctor functions in this module for each request so that the code in the client can be clearer, e,g,, `pub fn new_get_request(key: impl Into<Key>, ...) -> kvrpcpb::RawGetRequest`

Reply:
seems good